libs/wingsdk/src/target-tf-aws/platform.ts

```diff
@@ -87,7 +87,7 @@ export class Platform implements IPlatform {
         await client.send(
           new UpdateSecretCommand({
             SecretId: name,
-            SecretString: JSON.stringify({ [name]: value }),
+            SecretString: value,
```

Comment:
Contemplating the best way to test this.

Reply:
how was `JSON.stringify({ [name]: value })` tested?